src/coprocessor/tracker.rs

```diff
@@ -140,6 +141,7 @@ impl Tracker {
             self.total_perf_stats += perf_stats.delta();
         }
         self.current_stage = TrackerState::ItemFinished;
+        // TODO: Need to record time between Finish -> Begin Next?
```

Comment:
It's another kind of "wait time" for unary requests.

Reply:
Seems "wait time" is ambiguous. `time between Finish -> Begin Next` also has different meanings for unary requests and stream requests.

Reply:
accumulated_yatp_switch_time? (But I don't have plans to add it in this PR)